src/defaultsettings.cpp

```diff
@@ -323,6 +323,8 @@ void set_default_settings(Settings *settings)
 	// Server
 	settings->setDefault("disable_escape_sequences", "false");
 	settings->setDefault("strip_color_codes", "false");
+	settings->setDefault("stacksize_default_node", "99");
+	settings->setDefault("stacksize_default_item", "99");
```

Comment:
You don't need to add this to defaultsettings.cpp because it is only used in the lua files,
and maybe the settings have to be converted to a number.
```lua
tonumber(minetest.settings:get("stacksize_default_node")) or 99
```